src/compile/legend/encode.ts

```diff
@@ -87,19 +84,11 @@ export function symbols(fieldDef: FieldDef, symbolsSpec: any, model: UnitModel,
   }
 
   if (channel !== SHAPE) {
-    if (legend.symbolShape !== undefined) {
-      symbols.shape = {value: legend.symbolShape};
-    } else if (cfg.point.shape !== undefined) {
+    if (cfg.point.shape !== undefined) {
```

Comment:
1) Do we need to set symbols.shape to `config.point.shape`?  (I guess we do, but please test.)

2) I think we have to handle the case when users specify `encoding.shape.value`.